web/handlers.go

```diff
@@ -176,16 +176,13 @@ func (h Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		// Instruct the browser not to display us in an iframe unless is the same origin for anti-clickjacking
 		w.Header().Set("X-Frame-Options", "SAMEORIGIN")
 
-		// Add unsafe-eval to the content security policy for faster source maps in development mode
 		devCSP := ""
-		if model.BuildNumber == "dev" {
-			devCSP += " 'unsafe-eval'"
-		}
-
-		// Add unsafe-inline to unlock extensions like React & Redux DevTools in Firefox
-		// see https://github.com/reduxjs/redux-devtools/issues/380
-		if model.BuildNumber == "dev" {
-			devCSP += " 'unsafe-inline'"
+		// Add flags like unsafe-eval and unsafe-inline for debugging during development
```

Comment:
@agnivade, is there a recommended Go library we should use for parsing the desired flags as a string vs. an array? I seem to recall your mentioning we can copy what Go does in a case like this.

Reply:
Yes, I'll need to check the code. Although I don't think it's available as a library. We just need to copy over the `GODEBUG` parsing implementation.

Reply:
@agnivade I'm very new to golang. Can you please tell me when I can find the `GODEBUG` parsing implementation.

Reply:
It's nothing too complicated. You just have to run a loop on `strings.Split(os.Getenv("GODEBUG"), ",")`. And then split again on "=", and check for the appropriate key.